.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```

Comment:
it's better to add `.vscode` to `.gitignore` and remove this file from the commit